culture.qmd

```diff
@@ -142,23 +137,39 @@ As a Ph.D. student, you're expected to ...
 - Work with the PI to make short- and long-term research, professional, and career plans to stay on track towards your next steps.
 
 ## Undergraduates (Short-term interns/Visiting scientists)
--   **Be present.** Attend meetings and bring forth your best effort.
--   **Ask for help.** We encourage you to be proactive; ask questions and provide any updates or ideas to our Slack channels.
--   **Strive for growth.** It will be challenging, but we want you to keep trying and persist – learning is the main goal, and being receptive and open to constructive feedback is a great way to improve.
--   **Give yourself grace.** You’ve earned a spot in our community, and you belong here. Failure is inevitable with the work we’re doing; your work doesn’t define you or your abilities. We're all here for you!
--   **Stay organized.** Keeping track of your priorities, tasks, and responsibilities will allow you to monitor your progress and make it easier for others to help you. Time management is essential for pacing yourself to achieve your goals.
--   **Collaborate.** We encourage you to present your work and discuss it with others frequently. Learn about the projects your labmates are working on, communicate any troubles or issues before they boil (work-related and not), and keep a positive mental attitude!
--   **Own your research.** This is *your* research process, so be sure that you’re putting forth your best work and that you continue to practice scientific reading and writing.
--   **Take advantage of resources.** There are many avenues for learning in research. Here are a few to get you started:
+-   __Be present.__ Attend meetings and bring forth your best effort.
```

Comment:
No, no issues there -- it's just unifying so it's the same format as the italicized points earlier in the same document. I figured it was better if both were either italicized or bolded and went with __the former__ solely because it came first. I can revert if preferred.